src/site/apt/sessions/session-access-grants.apt.vm

```diff
@@ -3,6 +3,124 @@ Access Grants
     {{{https://docs.inrupt.com/ess/latest/security/access-requests-grants/}Access Grants}} provide an alternative way to give access to your Solid resources. The Inrupt Java Client Libraries
     provide tooling for working with Access Grants.
 
+    To use {{{https://docs.inrupt.com/ess/latest/security/access-requests-grants/}Access Grants}}, the resources one wants to access must have {{{https://docs.inrupt.com/ess/latest/security/access-requests-grants/#access-grants-acp}ACP that enables the use of access grants}} and the Authorization Service must be able to exchange an access grant for a token (for example such as a {{{https://docs.inrupt.com/ess/latest/services/service-uma/}UMA Service}}).
+
+* ACP that enables the use of Access Grants
+
+    One needs to edit the ACP of the resource the access grant will be used for. 
+    
+** Manually adding triples to ACP
+    
+    This can be done manually through using a resource management interface such as {{{https://podpro.dev/} Pod Pro}}.
+
+    The next example, adds a new access control that matches for <<<http://www.w3.org/ns/solid/vc#SolidAccessGrant>>>> and allows Read and Write for a Solid Container and its contents. 
+
++---
+prefix acp: <http://www.w3.org/ns/solid/acp#>
+prefix acl: <http://www.w3.org/ns/auth/acl#>
+
+# ACR for a Solid Container
+<>
+        a acp:AccessControlResource ;
+        # Add access control for the container itself
+        acp:accessControl <#newAccessControl> ;
+        # Add access control for contained resources
+        acp:memberAccessControl <#newAccessControl> .
+
+<#newAccessControl>
+        a acp:AccessControl ;
+        # The access control applies one policy
+        acp:apply <#newPolicy> .
+
+<#newPolicy>
+        a acp:Policy ;
+        # The policy requires one matcher to be satisfied
+        acp:allOf <#newMatcher> ;
+        # The policy grants read and write when its matcher is satisfied
+        acp:allow acl:Read, acl:Write .
+
+<#newMatcher>
+        a acp:Matcher ;
+        # The matcher is satisfied if a VC of type Solid Access Grant was presented
+        acp:vc <http://www.w3.org/ns/solid/vc#SolidAccessGrant> .
++---
+
+** Adding triples to ACP in code
+
+    One can add the above mentioned triples to an ACP using code. The next example will add a new access control that matches for <<<http://www.w3.org/ns/solid/vc#SolidAccessGrant>>> and allows Read and Write on a Solid resource the ACP belongs to. We are using only the <<<http://www.w3.org/ns/solid/acp#accessControl>>> predicate in the <<<AccessControlResource>>>.
```

Comment:
```suggestion
    One can write the above triples to an ACL auxiliary resource using code. The next example creates an access control that matches for <<<http://www.w3.org/ns/solid/vc#SolidAccessGrant>>> and allows Read and Write over a Solid Resource. We are using only the <<<http://www.w3.org/ns/solid/acp#accessControl>>> predicate in the <<<AccessControlResource>>> so contained resources are not affected.
```

Reply:
This code seems to override an ACR (as opposed to add to).
Or is it that authClient.update is using a patch? Or adding triples somehow?

Reply:
Good point. I first will try to collect the existing triples and than append the new ones.